bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
Thing is, we (almost) never use the items or values or keys functions to access the list directly, we always use it for iteration.

I have started to use items() instead of iteritems() exclusively since it is python3 compatible syntax and frankly, for the list sizes I am dealing with building the list in memory doesn't really matter.

As such, converting all instances of items() to [items()] is absolutely not what I want. My guess would be that 100% of items() calls can stay items() calls.